File: examples/t1.py
from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister

q = QuantumRegister(2, "q")
c = ClassicalRegister(2, "c")
qc = QuantumCircuit(q, c)

qc.reset(q)
qc.x(q[0])
qc.x(q[1])
qc.delay(1, q[0])
qc.delay(1, q[1])
qc.measure(q, c)

File: examples/t1.cpp
sim.set_num_qubits(2);
std::vector<int> creg(2);
// Unsupported operation: reset
// Unsupported operation: reset
sim.gate_x(0, {}, {});
sim.gate_x(1, {}, {});
// Unsupported operation: delay
// Unsupported operation: delay
creg[0] = sim.measure(0);
creg[1] = sim.measure(1);
